nestedtensor/nested/nested.py

```diff
@@ -282,7 +282,7 @@ def to_list(self):
     def to_tuple(self):
         return self._impl.to_tuple()
 
-    def to_tensor_mask(self, mask_dim=None):
+    def to_tensor_mask_tuple(self, mask_dim=None):
         """Returns a named tuple TensorMask with two tensors (tensor, mask)
```

Comment:
You should update this unless it still fully applies in context of this PR to keep track of our semantics.